### DISCUSSION TITLE
File > Open model from file: greyed out?

### DISCUSSION THREAD

**Discussion by bluehenve** [TabularEditor/TabularEditor3#179] [Other Topics]
Why is this menu item greyed out: File > Open model from file
I have the version 3.0.10 Desktop Edition
Regards, Henrik

**Comment by otykier** [comment]
As per our licensing terms, the Desktop Edition of Tabular Editor 3 can only be used as an External Tool for Power BI Desktop.

https://docs.tabulareditor.com/projects/te3/en/latest/editions.html

**Reply by otykier** [reply]
To clarify: The Desktop Edition is limited to only allow access to a data model (.pbix or .pbit file) loaded in Power BI Desktop. Microsoft unfortunately does not offer a supported way to load .pbix or .pbit files directly in 3rd party tools, which is why the file has to be opened through Power BI Desktop.

**Comment by bluehenve** [comment]
Okay, thanks for the quick reply
